moving_arm movingArmData
fixed_arm  fixedArmData